web/app/assets/locales/locale-tr.js

```diff
@@ -32,7 +32,7 @@
         pending: "sırada %(blocks)s blok var"
     },
     propose: "teklif et",
-    cancel: "iptal et",
```

Comment:
Hey, does that every time  happen with the same comma ? that is what I download from transifex after making changes. 


Have a great day !

Mithat Arslan
mithars@gmail.com



> On 25Nov, 2015, at 11:15 AM, Sigve Kvalsvik <notifications@github.com> wrote:
> 
> In web/app/assets/locales/locale-tr.js <https://github.com/cryptonomex/graphene-ui/pull/532#discussion_r45843504>:
> 
> > @@ -32,7 +32,7 @@
> >          pending: "sırada %(blocks)s blok var"
> >      },
> >      propose: "teklif et",
> > -    cancel: "iptal et",
> You have to keep this comma, if not the file is not valid JSON.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cryptonomex/graphene-ui/pull/532/files#r45843504>.
>